### PR TITLE
AR noise better docstring

### PR DESCRIPTION
**Pull request by behinger**

### CODE REVIEW

**Comment by jschepers** [comment]
Thanks for rewriting the doc string. I have two things that could be added:
- N is the length of the simulated signal in samples, right? I think this is at the moment only implicitly mentioned.
- Maybe you could write "autoregressive (AR)" instead of only "AR" because there might be users who don't know what the abbreviation stands for.

**Comment by jschepers** [comment]
I would suggest merging your changes in the `joss-doc-suggestions` branch so that all changes related to the JOSS docs review are in the same place. But I don't know whether it's easily possible to change this in the existing PR. But since it's only a small change it might not matter too much.

File: src/noise.jl [hunk 1]
```diff
@@ -72,7 +72,7 @@ Noise with exponential decay in AR spectrum.
 `noiselevel` is used to scale the noise
 
 !!! warning
-    Current implementation: Cholesky of NxN matrix needs to be calculated, which might need lots of RAM.
+    With the current implementation we try to get exponential decay over the whole AR spectrum, which is N-Samples long. This involves the inversion of a cholesky matrix of size NxN matrix, which will need lots of RAM for non-trivial problems.
```
Comment (behinger) [inline src/noise.jl:75]:
```suggestion
    With the current implementation we try to get exponential decay over the whole autocorrelation (AR) spectrum, which is N-Samples (the total number of samples) long. This involves the inversion of a cholesky matrix of size NxN matrix, which will need lots of RAM for non-trivial problems.
```

File: src/noise.jl [hunk 2]
```diff
@@ -72,7 +72,7 @@ Noise with exponential decay in AR spectrum.
 `noiselevel` is used to scale the noise
 
 !!! warning
-    Current implementation: Cholesky of NxN matrix needs to be calculated, which might need lots of RAM.
+    With the current implementation we try to get exponential decay over the whole autocorrelation (AR) spectrum, which is N-Samples (the total number of samples) long. This involves the inversion of a cholesky matrix of size NxN matrix, which will need lots of RAM for non-trivial problems.
```
Comment (behinger) [inline src/noise.jl:75]:
```suggestion
    With the current implementation we try to get exponential decay over the whole autoregressive (AR) spectrum, which is N-Samples (the total number of samples) long. This involves the inversion of a cholesky matrix of size NxN matrix, which will need lots of RAM for non-trivial problems.
```

**Comment by behinger** [comment]
done :)

File: src/noise.jl [hunk 3]
```diff
@@ -72,7 +72,7 @@ Noise with exponential decay in AR spectrum.
 `noiselevel` is used to scale the noise
 
 !!! warning
-    Current implementation: Cholesky of NxN matrix needs to be calculated, which might need lots of RAM.
+    With the current implementation we try to get exponential decay over the whole autoregressive (AR) spectrum, which is N-Samples (the total number of samples) long. This involves the inversion of a cholesky matrix of size NxN matrix, which will need lots of RAM for non-trivial problems.
```
Comment (jschepers) [inline src/noise.jl:75]:
```suggestion
    With the current implementation we try to get exponential decay over the whole autoregressive (AR) spectrum, which is N samples (the total number of samples in the signal) long. This involves the inversion of a Cholesky matrix of size NxN matrix, which will need lots of RAM for non-trivial problems.
```

**Pull request merged**